src/documents/TextToSpeechAPI.md

```diff
@@ -0,0 +1,580 @@
+            </td>
+        </tr>
+        <tr>
+            <th>Type</th>
+            <td>default</td>
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>
+                none
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### `onPause` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                This event fires when the text is paused whilst being spoken.
+            </td>
+        </tr>
+        <tr>
+            <th>Type</th>
+            <td>default</td>
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>
+                none
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### `onResume` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                This event fires when a paused text resumes being spoken.
+            </td>
+        </tr>
+        <tr>
+            <th>Type</th>
+            <td>default</td>
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>
+                none
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### `onError` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                This event fires when an error occurs that prevents the text from being spoken.</em></p>
+            </td>
+        </tr>
+        <tr>
+            <th>Type</th>
+            <td>default</td>
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>
+                none
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### `onSpeechQueued` ###
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                This event fires when a text has been queued up to be spoken.
+            </td>
+        </tr>
+        <tr>
+            <th>Type</th>
+            <td>default</td>
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>
+                <dl>
+                    <dt>text</dt>
+                    <dd>
+                        The text that has been added to the queue.
+                    </dd>
+                </dl>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## `utteranceOpts` Options ##
+
+_**Note:** If needed, please read the [Component Configuration Options](ComponentConfigurationOptions.md) document for a full description of infusion component options._
+
+The only option supported by the Text To Speech component is `utteranceOpts`This option is a javascript object that contains a number of attributes that users can use to define the behaviour of their `SpeechSynthesisUtterance` instances. [The `SpeechSynthesisUtterance` instance](https://dvcs.w3.org/hg/speech-api/raw-file/tip/speechapi.html#utterance-attributes) is a part of the web speech API that the Text To Speech component interacts with. These attributes include:
```

Comment:
I might reword this as

The only option supported by the Text To Speech component is `utteranceOpts`This option is a javascript object that contains attributes that users can use to define the behaviour of the [`SpeechSynthesisUtterance` instance](link here) (a part of the web speech API that the Text To Speech component interacts with). These attributes include:

How does that look?